packages/main/src/themes/Panel.css

```diff
@@ -16,20 +19,18 @@
 }
 
 .ui5-panel-header {
-	height: var(--_ui5_panel_header_height);
+	min-height: var(--_ui5_panel_header_height);
```

Comment:
min-height results in 2.75 rem for Cozy, should be 2.5 for Horizon and Fiori 3 (both Cozy and Compact)